Kinvey/Kinvey/FindOperation.swift

```diff
@@ -102,17 +102,18 @@ internal class FindOperation<T: Persistable>: ReadOperation<T, AnyRandomAccessCo
     
     private func fetchAutoPagination(multiRequest: MultiRequest, count: Int) -> Promise<AnyRandomAccessCollection<T>> {
         return Promise<AnyRandomAccessCollection<T>> { fulfill, reject in
-            let nPages = Int64(ceil(Double(count) / Double(MaxSizePerResultSet)))
+            let maxSizePerResultSet = options?.maxSizePerResultSet ?? MaxSizePerResultSet
```

Comment:
Corner case, but does zero cause a problem?

Reply:
good question, also negative values should not be allowed. I will add some protection to avoid this scenario